fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOFiber.scala

```diff
@@ -20,11 +21,19 @@ object BIOFiber3 {
   implicit final class ToCats[FR[-_, +_, +_], A](private val bioFiber: BIOFiber3[FR, Throwable, A]) extends AnyVal {
     def toCats(implicit F: BIOFunctor3[FR]): cats.effect.Fiber[FR[Any, Throwable, ?], A] = new cats.effect.Fiber[FR[Any, Throwable, ?], A] {
       override def cancel: FR[Any, Throwable, Unit] = F.void(bioFiber.interrupt)
+
       override def join: FR[Any, Throwable, A] = bioFiber.join
     }
   }
+
 }
 
 object BIOFiber {
   @inline def fromZIO[E, A](f: Fiber[E, A]): BIOFiber3[ZIO, E, A] = BIOFiber3.fromZIO(f)
+  @inline def fromMonix[E, A](f: bio.Fiber[E, A]): BIOFiber[bio.IO, E, A] = new BIOFiber[bio.IO, E, A] {
+    override def join: bio.IO[E, A] = f.join
+    // I could perform  only cancel
+    override def observe: bio.IO[Nothing, BIOExit[E, A]] = ???
```

Comment:
@VladPodilnyk I think `observe` may be implementable as `fiber.join.sandboxBIOExit`

Reply:
Done.